datalad/support/tests/test_external_versions.py

```diff
@@ -231,9 +232,15 @@ class thing_with_list_version:
 
 
 def test_system_ssh_version():
+    try:
+        WitlessRunner().run(['ssh', '-V'], protocol=StdOutErrCapture)
+    except FileNotFoundError as exc:
+        raise SkipTest(f"no ssh binary available: {exc}")
```

Comment:
I think it would be preferrable to use [`pytest.skip()`](https://docs.pytest.org/en/7.3.x/reference/reference.html#pytest-skip) instead.